x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -620,19 +620,21 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
     });
   }, [updatedAt, showUpdating, timelinesUi]);
 
+  const allAlertTableFilters = useMemo(() => alertMergedFilters, [alertMergedFilters]);
   const renderGroupedAlertTable = useCallback(
     (groupingFilters: Filter[]) => {
+      allAlertTableFilters.push(...groupingFilters);
       return (
         <AlertsTableComponent
           configId={ALERTS_TABLE_REGISTRY_CONFIG_IDS.RULE_DETAILS}
           flyoutSize="m"
-          inputFilters={[...alertMergedFilters, ...groupingFilters]}
```

Comment:
New array reference on every render of this component. But may not even be properly memoized anyways since it's a render function? Need to further investigate